uber/templates/jobs/staffers.html

```diff
@@ -49,6 +49,7 @@
     <th> Hours Total </th>
     <th> Checked In</th>
 </tr>
+</thead>
```

Comment:
This was an unrelated fix.  @Lusarien noticed that ``datatables`` was putting icons in the wrong place, and then figured out that it was because we weren't closing the ``<thead>``.